clients/unshaded/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -265,7 +269,11 @@ public synchronized void close() throws IOException {
    * @throws IOException if the underlying master RPC fails
    */
   synchronized void completeFile(long fid) throws IOException {
-    mFSMasterClient.completeFile(fid);
+    try {
+      mFSMasterClient.completeFile(fid);
+    } catch (TException e) {
```

Comment:
I figured since it's deprecated, it's not worth throwing all the different types of exceptions, and propagating that everywhere TachyonFS is used. Do you think it's still worthwhile?